From 18bafa0443c975d9da7664cf6dbae323cd8cb9e3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Olivier=20Cr=C3=AAte?= <olivier.crete@collabora.co.uk>
Date: Fri, 8 Oct 2010 12:41:10 -0400
Subject: [PATCH] rtpconference: The ptime/maxptime in caps are actually uints, not strings

---
 gst/fsrtpconference/fs-rtp-conference.c |    9 +++++++--
 1 files changed, 7 insertions(+), 2 deletions(-)

diff --git a/gst/fsrtpconference/fs-rtp-conference.c b/gst/fsrtpconference/fs-rtp-conference.c
index 838dd25..2498970 100644
--- a/gst/fsrtpconference/fs-rtp-conference.c
+++ b/gst/fsrtpconference/fs-rtp-conference.c
@@ -910,8 +910,13 @@ fs_codec_to_gst_caps (const FsCodec *codec)
        item = g_list_next (item)) {
     FsCodecParameter *param = item->data;
     gchar *lower_name = g_ascii_strdown (param->name, -1);
-    gst_structure_set (structure, lower_name, G_TYPE_STRING, param->value,
-      NULL);
+
+    if (!strcmp (lower_name, "ptime") || !strcmp (lower_name, "maxptime"))
+      gst_structure_set (structure, lower_name, G_TYPE_UINT,
+          atoi (param->value), NULL);
+    else
+      gst_structure_set (structure, lower_name, G_TYPE_STRING, param->value,
+          NULL);
     g_free (lower_name);
   }
 
